scripts/averaging/create_climo_files.py

```diff
@@ -205,7 +214,9 @@ def process_variable(ts_files, syr, eyr, output_file):
         cam_ts_data.assign_coords(time=time)
         cam_ts_data = xr.decode_cf(cam_ts_data)
     #Extract data subset using provided year bounds:
-    cam_ts_data = cam_ts_data.sel(time=slice(syr, eyr))
+    #cam_ts_data = cam_ts_data.sel(time=slice(syr, eyr))
+    tslice = get_time_slice_by_year(cam_ts_data.time, int(syr), int(eyr))
+    cam_ts_data = cam_ts_data.isel(time=tslice)
```

Comment:
Done.

This is probably my sloppy branch management. I think I shared this code with Justin offline, so it could exist in a separate PR.

I will leave it here, since I think it works. Be on the look out for similar code coming in another PR at some point.